.gitignore

```diff
@@ -41,3 +41,6 @@ htmlcov
 .settings
 venv
 .vscode
+
+**/__pycache__
+**/generated-sources
```

Comment:
Are we planning to include generated sources in this repo to avoid users having to run antlr themselves?

Reply:
Generated sources are included already. This path is no longer used and may be removed from .gitignore

Reply:
Removed